mep/people/templates/people/member_list.html

```diff
@@ -17,9 +17,26 @@
     <form id="members-form">
         {% render_field form.query %}
         <label for="{{ form.query.id_for_label }}"></label>
+        {% if form.errors %}
+        <div role="alert" class="errors" aria-labelledby="errors" aria-live="polite">
+            <span id="errors">Errors:</span>
+            <ul>
+            {% for field, errors in form.errors.items %}
+            {# no clean way to get field labels here, so we just collapse the error lists #}
+            {% for error in errors %}
```

Comment:
Would be more readable if the nested for loop was indented. Thanks for documenting why we're not displaying any field labels